trackpy/diag.py

```diff
@@ -20,3 +21,35 @@ def performance_report():
         print("FAST: Using pyfftw for image preprocessing.")
     else:
         print("SLOW: pyfftw not found (slower image preprocessing).")
+
+
+def dependencies():
+    """
+    Give the version of each of the dependencies -- useful for bug reports.
+
+    Returns
+    -------
+    result : dict
+        mapping the name of each package to its version string or, if an
+        optional dependency is not installed, None
+    """
+    packages = ['six', 'numpy', 'scipy', 'matplotlib', 'pandas',
+                'scikit-image', 'pyyaml', 'pytables', 'numba', 'pyfftw']
+    result = OrderedDict()
+    for package_name in packages:
+        try:
+            package = __import__(package_name)
```

Comment:
Good point. (To explain, we just learned this behaves better, though not sure why.)

Done. Tests pass and behavior looks the same.